api/src/main/java/com/basistech/rosette/api/RosetteAPI.java

```diff
@@ -124,7 +125,8 @@
      * @throws IOException         General IO exception
      */
     public RosetteAPI(String key) throws IOException, RosetteAPIException {
-        this(key, null);
+        this(key, DEFAULT_URL_BASE);
+        checkVersionCompatibility();
```

Comment:
I think we still want to check version compatibility with alt url, so this should relocate back to the other cstor.